op-acceptance-tests/tests/interop/loadtest/interop_load_test.go

```diff
@@ -68,19 +68,19 @@ func TestLoad(gt *testing.T) {
 }
 
 func SpamInteropTxs(t devtest.T, numInitTxs uint64, source *L2, dest *L2, supervisor *dsl.Supervisor) {
-	var wg sync.WaitGroup
-	defer wg.Wait()
+	var relayWg sync.WaitGroup
+	defer relayWg.Wait()
 	msgsCh := make(chan []types.Message, 100)
 	defer close(msgsCh)
```

Comment:
The correctness of function is very sensitive to the location of this statement. If this defer were placed before the `defer relayWg.Wait()` line there's be a problem because the `for msgs := range msgsCh {}` loop would exit immediately. Might be a bit safer to close the messages channel at the end of the function after the `for i := range numInitTxs {}` loop.

Reply:
We do risk the channel not being cleaned up in that case (not a big problem). I'll add a comment.